Makefile

```diff
@@ -7,7 +7,7 @@ install: prepare
 prepare:
 	python3 -m pip install cryptography tox
 	mkdir -p dev-env/certs
-	python3 run_server.py --gen-certs
+	python3 dev-env/config/dev-utils.py gen-certs
 
 lint:
 	tox -e lint types
```

Comment:
Also, `make docs` and `make release` do not work.

should  it be?

```toml
.PHONY: docs all release 
```
instead of 
```toml
.PHONY: all release 
```

and 

```toml
docs:
       tox -e docs

release:
       tox -e release
```

instead of

```toml
docs: tox -e docs                                                                                                                           
                                                                                                                                            
release: tox -e release  
```

Reply:
And I had a look, in dev section of README we have no `make lint` and other commands and this PR is about making the dev part works, so if you are okey, let's submit another PR for this. what do u think?

Reply:
It should be `tox -e lint,types`

Reply:
do i have to change it here or in another PR?

Reply:
No, preference!